tools/ui-components/src/image/Image.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { Story } from '@storybook/react';
+import React from 'react';
+import { Image, ImageProps } from './Image';
+
+const story = {
+  title: 'Example/Image',
+  component: Image
+};
+
+const Template: Story<ImageProps> = args => {
+  return <Image {...args} />;
+};
+const baseArgs = {
+  alt: '',
+  src: ''
+};
+
+export const Simple = Template.bind({});
```

Comment:
more descriptive name should be used. If this is the default version, it can be named default.